src/main/kotlin/ar/edu/unahur/obj2/socios/Barrio.kt

```diff
@@ -0,0 +1,21 @@
+package ar.edu.unahur.obj2.socios
+
+abstract class Barrio {
+    abstract fun influirPropina(cliente: Cliente, propinaSegunAnimo : Int): Int
+}
+
+object LasRosas : Barrio() {
+    override fun influirPropina(cliente: Cliente, propinaSegunAnimo : Int) = 50
+}
+
+object LasLauchas : Barrio() {
+    override fun influirPropina(cliente: Cliente, propinaSegunAnimo : Int) = -propinaSegunAnimo / 2
```

Comment:
Por qué `-`? 
> **Las Lauchas** 🐭: reduce a la mitad la propina
```suggestion
    override fun influirPropina(cliente: Cliente, propinaSegunAnimo : Int) = propinaSegunAnimo / 2
```
Se supone que esto va a devolver la mitad de lo que reciba por parametro.

Reply:
Le mandamos así "-",  porque le restábamos la mitad de la propina después. Malísimo igual, pero fue lo único que se nos ocurrió. 😢 😢